src/changes-backwards-compatible.apib

```diff
@@ -4,6 +4,10 @@
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
+#### Latest
+
+- We added filtering on a new `link` type `deal` to `events.list`.
```

Comment:
Filtering is not the main takeaway I think. Might be better to word this as:
```suggestion
- As events can be linked to deals, we added the `deal` link type to `events.list` and `events.info`, and allow filtering on deals in `events.list`.
```

Reply:
I will use your description but not mention the info as we have not added that yet.